src/dtx/dtx_common.c

```diff
@@ -155,6 +147,8 @@ dtx_batched_commit(void *arg)
 	struct dtx_batched_commit_args	*dbca;
 	struct dtx_batched_commit_args	*tmp;
 
+	dmi->dmi_dtx_batched_started = 1;
```

Comment:
Optional: Instead of introducing a new bit for every long run ULT, I think it's better to use the unified "sched_request". See the sched APIs like sched_req_sleep(), sched_req_wait(). Maybe we can do this cleanup in later patch.

Reply:
We plan to change the per xstream based batched commit ULT to per container based ULT. So current patch is a temporary solution.